src/Storage/Commands.Storage/File/Cmdlet/RemoveAzureStorageShareStoredAccessPolicy.cs

```diff
@@ -84,8 +84,6 @@ internal bool RemoveAzureShareStoredAccessPolicy(IStorageFileManagement localCha
         [PermissionSet(SecurityAction.Demand, Name = "FullTrust")]
         public override void ExecuteCmdlet()
         {
-            WriteWarning("Force parameter in this cmdlet will be removed in a future release.");
```

Comment:
Same comment - the Force parameter is still here, so let' shandle this seperataely